### PR TITLE
Add failing logic to AT export if AT credentials are invalid

### PR DESCRIPTION
**Pull request by perryr16**
#### Any background context you want to provide?
If a user's AT credentials were invalid the page would hang

#### What's this PR do?
Fails gracefully if credentials are invalid

#### How should this be manually tested?
Inventory List > selection > Actions > Export to Audit Template. Once with invalid credentials, and once with valid credentials

#### Screenshots (if appropriate)
Screenshot 2023-10-13 at 1 17 34 PM

### CODE REVIEW

File: docker-compose.dev.yml [hunk 1]
```diff
@@ -22,7 +22,6 @@ services:
     environment:
       - DJANGO_SETTINGS_MODULE=config.settings.docker_dev
       - DJANGO_LOG_LEVEL
-      - EEEJ_LOAD_SMALL_TEST_DATASET=True
```
Comment (perryr16) [inline docker-compose.dev.yml]: Should the docker-compose.dev.yml include this env var?

**Comment by nllong** [comment]
@perryr16  -- do you know why I would be getting this error? I know that my username and PW are right. Is there something to do with the org that I have?

**Comment by perryr16** [comment]
Are you using my org token? Is your `AUDIT_TEMPLATE_HOST` set to  `'https://buildingenergyscore.energy.gov'`?

File: docker-compose.dev.yml (hunk 1 above)
Comment (nllong) [reply docker-compose.dev.yml]: I think so because that is what we use for our GitHub regression testing, right? We want that to run faster.

**Review by nllong — APPROVED** [review]
worked! but note that we need to fix allowing passwords with `%` symbols.

**Pull request merged**